generate_data/n5-java/generate_data.sh

```diff
@@ -1,3 +1,7 @@
+# clean the n5 data. ideally, this would be done in n5-java, but I don't know if
+# it supports truncate mode
+rm -r data/n5-java.n5
```

Comment:
Could this go under the `cd` line, using the relative path `../../data/n5-java.n5`? Then it stays agnostic as to where it's called from.

Reply:
Yes, makes sense.